src/com/sahokia/kos/concerttickets/model/Admin.java

```diff
@@ -0,0 +1,21 @@
+package com.sahokia.kos.concerttickets.model;
+
+import com.sahokia.kos.concerttickets.service.TicketService;
+
+public class Admin extends User {
+
+
+    public void checkTicket(Ticket ticket, TicketService ticketService) {
```

Comment:
Why your 2nd param is ticketService, not ticketStorage? 
And in general - do the admin could manage a lot of storages?

Reply:
Add ticketStorage as a field for the admin and removed that param from the method